Assets/Scripts/PanZoom.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PanZoom : MonoBehaviour
+{
+	Vector3 touchFirst;
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    	if(Input.GetMouseButtonDown(0)){ //starts at the very beggining of the touch
```

Comment:
ah this might need to get moved into a `OnManipulationStarted(...)` event function, I can help you find an example somewhere else, likely the one inside of some of the snapcollider code